MetaMask/metamask-extension
# fix: add tooltip description to manageAccounts method

## **Description**

This PR adds the missing tooltip description to the `manageAccounts` permission.

~[Snap permissions source.](https://www.notion.so/bac3299d2c5241c599d2e5e7986e72f7?v=ef742a61bd844435b7171bd2e90b447e)~

Tooltip updated to: `Allow this Snap to add or remove Ethereum accounts, then transact and sign with these accounts.`

Review [comment](https://github.com/MetaMask/metamask-extension/pull/20964#issuecomment-1729886200) from @coreyjanssen 

## **Manual testing steps**

_1. Go to https://metamask.github.io/snap-simple-keyring/0.2.2/_
_2. Proceed to install snap using Flask_
_3. Hover tooltip icon. It should show the permission description_
_4. Install snap_
_5. Go to snap settings_
_6. See details of the manageAccounts permission. It should show the permission description_

## **Screenshots/Recordings**
_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

<img width="357" alt="Screenshot 2023-09-21 at 13 59 27" src="https://github.com/MetaMask/metamask-extension/assets/17601467/d8a71dbd-2d30-484b-9b4e-04d80ab23b42">

<img width="352" alt="Screenshot 2023-09-21 at 14 00 02" src="https://github.com/MetaMask/metamask-extension/assets/17601467/9e01d3a3-a475-4377-9cf5-384a388bf4c2">

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/52

## **Pre-merge author checklist**

- [x] I’ve indicated what issue this PR is linked to: Fixes https://github.com/MetaMask/accounts-planning/issues/52

## **Pre-merge reviewer checklist**
